src/runtime/createRunTime.tsx

```diff
@@ -1203,92 +1231,90 @@ function CreateRunTime({
                 value={containerImageSelected}
                 onChange={e => setContainerImageSelected(e.target.value)}
                 type="text"
-                placeholder="Enter URI,for example,gcr.io/my-project-id/my-image:1.0.1"
+                placeholder="Enter URI, for example,gcr.io/my-project-id/my-image:1.0.1"
               />
             </div>
             <div className="create-custom-messagelist">
-              {CUSTOM_CONTAINER_MESSAGE} </div>
-               <div className="create-container-message">
-                <div className="create-container-image-message">
+              {CUSTOM_CONTAINER_MESSAGE}{' '}
+            </div>
+            <div className="create-container-message">
+              <div className="create-container-image-message">
                 image. You must host your custom container on
```

Comment:
Please do not split the message this way. Either keep the entire message in a single constant and use </br> for new line or move  'image. You must host your custom container on' to another constant and make use of the constant here. You can rename the constants as CUSTOM_CONTAINER_MESSAGE_PART1 and USTOM_CONTAINER_MESSAGE_PART2